client/extensions/wp-job-manager/state/data-layer/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import { mergeHandlers } from 'state/action-watchers/utils';
+import { addHandlers } from 'state/data-layer/extensions-middleware';
+import settings from './settings';
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'wp-job-manager:errors' );
+const handlers = mergeHandlers( settings );
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15299/commits/1c66c0fe47182fca19f03dfe0664a4f4d0b17524.